### ISSUE TITLE
Unable to fetch additional props on fetchCalendars

### ISSUE THREAD

**Issue by m0dch3n** [natelindev/tsdav#137]
I added the following property when I fetch my calendars from Apple.
```typescript
    const props = {
...
      [`${DAVNamespaceShort.CALDAV_APPLE}:calendar-order`]: {},
...
    }
        const davCalendars = await davClient.fetchCalendars({ props })
```

However my davCalendars do then not include the calendarOrder.

How about adding the following PR: https://github.com/natelindev/tsdav/pull/136

So I could do:

```typescript
const davCalendars = await davClient.fetchCalendars({ props, projectedProps: { calendarOrder: true } })
```

**Comment by natelindev** [comment]
fetchCalendars right now do not have custom prop support, you can customize a function by combining other functions for now.
I will take a look at the PR and add this in the next release

**Comment by natelindev** [comment]
fixed in v2.0.5
